src/openforms/logging/logevent.py

```diff
@@ -478,38 +554,15 @@ def appointment_register_failure(appointment: AppointmentInfo, plugin, error):
         "appointment_register_failure",
         plugin=plugin,
         error=error,
+        tags=[TimelineLogTags.submission_lifecycle],
     )
 
 
 def appointment_register_skip(submission: Submission):
-    _create_log(submission, "appointment_register_skip")
-
-
-# - - -
-
-
-def appointment_update_start(appointment: AppointmentInfo, plugin):
-    _create_log(
-        appointment.submission,
-        "appointment_update_start",
-        plugin=plugin,
-    )
-
-
-def appointment_update_success(appointment: AppointmentInfo, plugin):
-    _create_log(
-        appointment.submission,
-        "appointment_update_success",
-        plugin=plugin,
-    )
-
-
-def appointment_update_failure(appointment: AppointmentInfo, plugin, error):
     _create_log(
-        appointment.submission,
-        "appointment_update_failure",
-        plugin=plugin,
-        error=error,
```

Comment:
Removed this since we do not support update in the new appointments flow (have to cancel first and create a new one).